### ISSUE TITLE
Add MathJax Support

### ISSUE THREAD

**Issue by kreynen** [CuBoulder/express#1817]
## Context

Evaluate https://www.drupal.org/project/mathjax

This was requested by Math in [EXP-8606].  When we looked at this 2? years ago, there were accessibility issues with the approaches we looked at.  Looking at https://www.mathjax.org/#features

> MathJax is compatible with screenreaders & provides zoom for everyone.

If meeting Math's needs is as simple as adding support for the MathJax library, this would be an easy feature to implement.  There would not be any shortcut to adding a formula via the WYSIWYG.  Users who wanted to take advantage of the feature would need to be able to write psuedo-code like...

`When \(a \ne 0\), there are two solutions to \(ax^2 + bx + c = 0\) and they are
  $$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$`

**Comment by ekellagher** [comment]
Information on Mathjax accessibility from Mathjax.org 
https://www.mathjax.org/#a11y
